code/__DEFINES/antagonists.dm

```diff
@@ -72,6 +72,7 @@
 
 
 // Heretic path defines.
+#define HERETIC "Heretic"
```

Comment:
This seems misplaced, as it's certainly not a path define
Move it above `// Heretic path defines.` and then give it its own comment `/// Generic heretic define used for trait sources` or w/e